From 6e10405aa25fe5e76b740d9ec59730e3f4470c7a Mon Sep 17 00:00:00 2001
From: Arseny Maslennikov <arseny@altlinux.org>
Date: Sun, 15 Oct 2023 11:00:00 +0300
Subject: [PATCH] seccomp: include `fchmodat2` in `@file-system`

Solus Note: This fixes a user-reported issue where Virtualbox VMs can't access host USB devices

Ref:
https://github.com/getsolus/packages/issues/1615
https://bugzilla.opensuse.org/show_bug.cgi?id=1219766#c20

---
 src/shared/seccomp-util.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/shared/seccomp-util.c b/src/shared/seccomp-util.c
index afe56d5167648..1712f3eff36a9 100644
--- a/src/shared/seccomp-util.c
+++ b/src/shared/seccomp-util.c
@@ -478,6 +478,7 @@ const SyscallFilterSet syscall_filter_sets[_SYSCALL_FILTER_SET_MAX] = {
                 "fchdir\0"
                 "fchmod\0"
                 "fchmodat\0"
+                "fchmodat2\0"
                 "fcntl\0"
                 "fcntl64\0"
                 "fgetxattr\0"
